runtime/jcl/common/stdinit.c

```diff
@@ -948,13 +948,22 @@ jobject getPropertyList(JNIEnv *env);
 jint
 completeInitialization(J9JavaVM * vm)
 {
+	jint result = JNI_OK;
 	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
-	vmFuncs->internalEnterVMFromJNI(vm->mainThread);
-	vmFuncs->sendCompleteInitialization(vm->mainThread, 0, 0, 0, 0);
-	vmFuncs->internalReleaseVMAccess(vm->mainThread);
-
-	if (vm->mainThread->currentException) {
-		return JNI_ERR;
+	J9VMThread *currentThread = vm->mainThread;
+	
+	vmFuncs->internalEnterVMFromJNI(currentThread);
+	vmFuncs->sendCompleteInitialization(currentThread, 0, 0, 0, 0);
+	vmFuncs->internalReleaseVMAccess(currentThread);
```

Comment:
This should be `internalExitVMToJNI` not `internalReleaseVMAccess`.  @gacholio Can you confirm?

Reply:
Gac confirmed the enter / exit need to be paired correctly

Reply:
Thanks @DanHeidinga fixing.

Reply:
For the record, it was broken before.

Reply:
And there are other places doing it, so I'd recommend leaving this one alone - we can revisit later on.

Reply:
The enter/exit calls are a bit of a cheat - they will work whether you're in the VM or not.

Reply:
Ok, will keep existing pair then. Created #2949 for future investigation.

Reply:
PR has been updated, please have another look.